examples/script/bzip2.yml

```diff
@@ -11,12 +11,8 @@ bzip_compression:
     dd if=/dev/zero of="$FILE" bs=1M count=1
     tar cvjf "$TARFILE" "$FILE"
     ls -l "$TARFILE"
-
-bzip_version:
```

Comment:
Why did you remove this test?

Reply:
it was getting a non-zero exit and was failing